scripts/node/icons/buildComponents.js

```diff
@@ -2,68 +2,44 @@
 const fs = require('fs');
 const path = require('path');
 const glob = require('glob');
-const svgr = require('@svgr/core').default;
+const toJsx = require('svg-to-jsx');
 const getComponentName = require('./helpers/getComponentName');
-const { svgoOptions, svgoPlugins } = require('./helpers/svgoConfig');
 const { buildConfig } = require('./icons.config');
 
-const FDS_HOC_NAME = 'withFdsIconWrapper';
-
-// copy our HOC for icon components to the build dir
-fs.copyFileSync(
-  buildConfig.react.template,
-  path.join(buildConfig.react.lib, `${FDS_HOC_NAME}.jsx`)
+// @function TEMPLATE
+const TEMPLATE = require('handlebars').compile(
+  fs.readFileSync(buildConfig.react.template).toString()
 );
 
 /**
- * SVGR does not provide a clean interface for customizing the output
- * of react components. Defining our own HOC allows us to customize things
- * like props for color and className.
+ * writes a react component to `lib/react`
+ * from an svg file
  *
- * @returns {String} file content for svgr-generated react component
- */
-const svgrTempalte = (
-  { template },
-  opts,
-  { imports, componentName, jsx }
-) => template.ast`${imports}
-import withFdsIconWrapper from './withFdsIconWrapper';
-const ${componentName} = (props) => ${jsx};
-export default withFdsIconWrapper(${componentName});
-`;
-
-/**
- * @param {String} iconName
- * @returns {Object} configuration for svgr
+ * @param {String} path to SVG file
  */
-const getSvgrConfig = (iconName) => ({
-  plugins: ['@svgr/plugin-svgo', '@svgr/plugin-jsx'],
-  ext: 'jsx',
-  prettier: false,
-  svgo: true,
-  svgoConfig: {
-    svgoOptions,
-    plugins: [...svgoPlugins, { cleanupIDs: { prefix: iconName } }],
-  },
-  expandProps: false,
-  template: svgrTempalte,
-});
+const writeComponent = async (filepath) => {
```

Comment:
optimized SVG from `lib` gets sanitized attributes for JSX, then interpolated with our component hbs template